### PR TITLE
Syndicate Surgery Tools

### PR DESCRIPTION
**Pull request by PopNotes**
:cl: PopNotes
add: The Syndicate surgery duffel bag now contains special Syndicate surgery tools that operate at THREE TIMES normal speed! They also look cooler and do a little more damage, as most Syndicate versions of items tend to do.
/:cl:

Maybe now someone will actually fucking buy this thing.

### CODE REVIEW

**Comment by PopNotes** [comment]
preview

**Comment by Arzion5** [comment]
Monomolecular is a meme. An actually monomolecular blade would turn to dust if it was made out of a solid, while trying to cut someone with it would cut so few molecules that by the time you finished swinging through someone they're already "stitched" and regenerated, even if you've slashed them in half or some shit.

I recommend adding syndicate drapes or merging all these tools into an omni-surgical tool, with the latter either being part of this PR or a different one. Syndicate drapes should be something that somehow either nullifies the chance of ever failing in a surgery, render items with only a chance of succeeding always succeeding, or unlock unique syndicate surgery interactions, such as brain-washing or direct upgrades.

**Comment by PopNotes** [comment]
this entire game is made of technobabble, there is no such thing as plasmic nanotubes

**Comment by Arzion5** [comment]
You are a technobabble >:(

File: hippiestation/code/modules/surgery/tools.dm [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	toolspeed = 0.3  
+
+/obj/item/circular_saw/syndicate
+	name = "syndicate circular saw"
+	desc = "Equipped with bluespace nanoemitters to part flesh and bone effortlessly."
+	icon = 'hippiestation/icons/obj/surgery.dmi'
+	icon_state = "ssaw"
+	lefthand_file = 'hippiestation/icons/mob/inhands/equipment/medical_lefthand.dmi'
+	righthand_file = 'hippiestation/icons/mob/inhands/equipment/medical_righthand.dmi'
+	force = 20
```
Comment (ghost) [inline hippiestation/code/modules/surgery/tools.dm]: Why are we adding 20 force damage item that delimbs and causes bleeding for 3TC which already is super powerful in terms of speed surgery?

**Comment by PopNotes** [comment]
Also, I wouldn't even know where to begin with the syndicate drapes idea, all normal drapes do is call attempt_initiate_surgery

File: hippiestation/code/modules/surgery/tools.dm (hunk 1 above)
Comment (PopNotes) [reply hippiestation/code/modules/surgery/tools.dm]: Lowered it to 20. Now it's a +3 damage boost from the normal saw/drill.

Comment (ghost) [reply hippiestation/code/modules/surgery/tools.dm]: It was 15 iirc

Comment (PopNotes) [reply hippiestation/code/modules/surgery/tools.dm]: I mean 18, my b.

**Comment by PopNotes** [comment]
code works, but I gotta wait for someone to approve it anyway

**Comment by calzilla1** [comment]
The new surgeey idea sounds pretty cool

**Comment by McBawbaggings** [comment]
I'm okay with it now, it was just not good to add a 20 brute damage weapon that crits in 5 hits in the uplink for 3TC and everything else it brings.

File: hippiestation/code/modules/surgery/tools.dm [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+	name = "syndicate cautery"
+	desc = "Uses state-of-the-art heating techniques."
+	icon = 'hippiestation/icons/obj/surgery.dmi'
+	icon_state = "scautery"
+	origin_tech = "materials=3;biotech=3;syndicate=3"
+	toolspeed = 0.3  
+
+/obj/item/surgicaldrill/syndicate
+	name = "syndicate surgical drill"
+	desc = "Outfitted with plasmic nanotubes in the bit and a stronger motor."
```
Comment (ghost) [inline hippiestation/code/modules/surgery/tools.dm]: stronger to what? It should be either strong or compared to something else.

Comment (PopNotes) [reply hippiestation/code/modules/surgery/tools.dm]: These are all implied to be modified versions of standard surgery tools, per all of their descriptions.

File: hippiestation/code/modules/surgery/tools.dm [hunk 3]
```diff
@@ -0,0 +1,56 @@
+/obj/item/retractor/syndicate
+	name = "syndicate retractor"
+	desc = "Reinforced with carbon-plasma alloy for a stronger grip."
```
Comment (ghost) [inline hippiestation/code/modules/surgery/tools.dm]: Comparative needs a comparison object again.

**Comment by PopNotes** [comment]
Am I a pretty boy now

**Comment by McBawbaggings** [comment]
the only thing "pretty" about you is being PRETTY GAY LMFAO

**Comment by McBawbaggings** [comment]
but yeah, descs look fine to me now

**Comment by PopNotes** [comment]
Shit and also fuck

**Pull request merged**